vulnerability-analyzer/src/main/java/org/hyades/processor/scanner/ScanProcessorSupplier.java

```diff
@@ -20,6 +20,8 @@ public interface ScanProcessorSupplier extends ProcessorSupplier<String, ScanTas
      */
     boolean isEnabled();
 
+    boolean isAliasSyncEnabled();
```

Comment:
Do we need this? It seems like all scanners pull this value from their respective `Config` class.